js/js-variables-and-numbers_simple-calculator/js/index.js

```diff
@@ -61,24 +73,36 @@ const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
 
 increaseByOneButton.addEventListener("click", () => {
   // 7. Increase the value of "operand1" by 1. Log the new value to the console.
+  const increaseByOne = operand1 + 1;
```

Comment:
In this case you would need to use `++` to make sure 1 is added every time you press the add 1 button, otherwise it will show always the same number, right?